### ISSUE TITLE
JetFormBuilder get ID of registered user

### ISSUE THREAD

**Issue by LevelUpGit** [CrocoBlock/suggestions#6699]
Add ability to get user ID from "Register user" action which is available when we select option "Add User ID to form data:" in "Insert/Update CCT", as of now there are only available fields that are in the form. But %user_id% and %inserted_cct_clanovi% are available in macro list for email action, it would be nice if they are available also in "Insert/Update CCT".

![Screenshot_1](https://user-images.githubusercontent.com/69863985/230131650-f4080513-3001-4820-be43-a3cbbdc9464e.jpg)
![Screenshot_2](https://user-images.githubusercontent.com/69863985/230131658-61e0f8f5-348d-4a3d-9c88-21510c12adb3.jpg)
![Untitsled](https://user-images.githubusercontent.com/69863985/230131663-61f9848c-fdcf-4bba-badc-619a61f85786.png)

Basically; add auto generated (User ID, Inserted CCT and rest) fields to selection in Insert/Update CCT, it would be useful if we are connecting that user with some CCT table.
